### PR TITLE
docs(upgrade): reminder to upgrade deis client

### PR DESCRIPTION
**Pull request by olalonde**

### CODE REVIEW

**Comment by Joshua-Anderson** [comment]
Deis no longer shows warnings when using an older client with an newer controller, since it should be safe.

That being said, upgrading for the new features is probably a good idea.

File: src/managing-workflow/upgrading-workflow.md [hunk 1]
```diff
@@ -173,3 +173,11 @@ deis-workflow-manager-2654760652-kitf9   1/1       Running   0          5m
 [configuring object storage]: ../installing-workflow/configuring-object-storage.md
 [keeper manifests]: http://helm-classic.readthedocs.io/en/latest/awesome/#keeper-manifests
 [minio]: https://github.com/deis/minio
+
+### Step 7: Upgrade the deis client
+
+Your deis platform users should now upgrade their deis client to avoid getting `WARNING: Client and server API versions do not match. Please consider upgrading.` warnings.
+
+```
+curl -sSL http://deis.io/deis-cli/install-v2.sh | bash && mv deis $(which deis)
```
Comment (Joshua-Anderson) [inline src/managing-workflow/upgrading-workflow.md]: You also want to `chmod +x deis`.

I might not try to move the cli. I put deis in `/user/local/bin`, so moving without sudo is going to fail.

**Comment by olalonde** [comment]
@Joshua-Anderson ah ok, I got the error with v2.2.0 client

File: src/managing-workflow/upgrading-workflow.md (hunk 1 above)
Comment (olalonde) [inline src/managing-workflow/upgrading-workflow.md]: I didn't have to `chmod +x deis`, I think the install script does it.

**Comment by olalonde** [comment]
@mboersma oops, should have been step #8 :( https://deis.com/docs/workflow/managing-workflow/upgrading-workflow/#step-7-upgrade-the-deis-client https://deis.com/docs/workflow/managing-workflow/upgrading-workflow/#step-7-verify-upgrade

**Pull request merged**